### PR TITLE
Fix extra landing space from header on smaller screens

### LINKED ISSUE #367: Extra landing space from header on smaller screens
**Describe the bug**
On smaller screens, switching app theme with the header expanded adds extra space to the landing as visible in the screenshot.

**To Reproduce**
1. Go to https://developerfolio.js.org/
2. Use a smaller viewport, around 375 pixels wide.
3. Click on header menu and switch theme.
4. Observe the extra 'padding' on top of the lander section.

**Expected behavior**
No extra space in between header and landing

**Is this responsiveness Issue**
Possibly yes

**Screenshots**
image

image

**Desktop (please complete the following information):**
 - Browser : Observed on Firefox, haven't checked Chrome
 - Version : 88

**Additional context**
We should also look into the extra absurd padding in between sections on smaller screens as visible below.

image

### PR DESCRIPTION
**Pull request by gajanandh**
Fixes #367

### CODE REVIEW

**Comment by kartikcho** [comment]
Hey @gajanandh, thanks for the fix! I think this should fix it but let's check the preview deployment too. (@saadpasta authorize the preview)
Also, I see changes to the icons in the public folder and they shouldn't be related to the changes in this PR, do you know what's up with them and can you revert it for this PR?

**Comment by kartikcho** [comment]
Also, updated the title and description to be more descriptive. (Using words like 'fixes' or 'addresses' will automatically close the associated issue too when this gets in)

You may add screenshots until the Preview deploy is up.

**Comment by gajanandh** [comment]
Hello @kartikcho , I have not  changed anything in public folder but somehow all the icons were staged for commit. Can you please tell me how to revert the changes



https://user-images.githubusercontent.com/80502737/123101398-a58f3f00-d451-11eb-8f99-a1eba5a20711.mp4

**Comment by kartikcho** [comment]
One way to do it would be to revert the first commit and then only stage `Header.scss`. Don't use `git add .` and check if the only file you need to change for this PR is staged with `git status`

File: src/components/header/Header.scss [hunk 1]
```diff
@@ -130,7 +136,7 @@
 }
 
 .header .menu-btn:checked ~ .menu-icon .navicon {
-  background: transparent;
+  background: transparent !important;
```
Comment (kartikcho) [inline src/components/header/Header.scss:139]: Any particular reason to set this as important?

Comment (gajanandh) [reply src/components/header/Header.scss:139]: I did that to fix the menu icon button in dark mode

**Comment by kartikcho** [comment]
@gajanandh the fix works, good job! Just revert the extra files committed in this PR and we should be good to go.

Also (unrelated to this PR), the header text doesn't use the old font anymore and I see the body font has been changed too, do you know about this change @naveen521kk @saadpasta?

**Comment by gajanandh** [comment]
I tried to revert commit But I am not able to do it. It will helpful If you could elaborate how to do it

**Comment by gajanandh** [comment]
I have reverted and commited only changes to Header.scss

**Review by kartikcho — APPROVED** [review]
Looks good, thanks!

**Comment by kartikcho** [comment]
@allcontributors please add @gajanandh for bug fixes.

**Comment by naveen521kk** [comment]
> Also (unrelated to this PR), the header text doesn't use the old font anymore and I see the body font has been changed too, do you know about this change @naveen521kk @saadpasta?

Can you create a new issue about this?

**Pull request merged**